### PR TITLE
feat(translation): add german language

### LINKED ISSUE #235: Add German translations
I would do that! 🙋🏼‍♂️

### PR DESCRIPTION
**Pull request by tautf**
Hey guys,

here is the PR for german translation. Closes #235 

Not sure if anyone else is from germany who could review.

Thank you guys 🎉

### CODE REVIEW

**Comment by FlorianTaut** [comment]
@lex111 i would love it if you could test this translation for me.
As is mentioned i have no internet at home currently 😢

**Comment by FlorianTaut** [comment]
Please set this on hold, i found a string which is constantly written and not translated, i want to send a pr for that before.

File: src/locale/languages/de.js [hunk 1]
```diff
@@ -0,0 +1,345 @@
...
+    },
+    pullDiff: {
+      title: 'Diff',
+      numFilesChanged: '{{numFilesChanged}} Dateien',
+      new: 'NEU',
+      deleted: 'GELÖSCHT',
+      fileRenamed: 'Datei ohne Änderungen umbenannt',
+    },
+    readMe: {
+      noReadMeFound: 'Keine README.md Datei gefunden.',
```
Comment (chinesedfan) [inline src/locale/languages/de.js]: @FlorianTaut  Please translate `readMeActions` as well.

Comment (FlorianTaut) [reply src/locale/languages/de.js]: @chinesedfan Good catch! Completely missed that one. Should be fixed now! 😄

File: src/locale/languages/de.js [hunk 2]
```diff
@@ -0,0 +1,347 @@
...
+  auth: {
+    login: {
+      connectingToGitHub: 'Verbinde mit GitHub...',
+      preparingGitPoint: 'Bereite GitPoint vor...',
+      cancel: 'ABBRECHEN',
+      welcomeTitle: 'Willkommen bei GitPoint',
+      welcomeMessage:
+        'Der umfangreichste GitHub Client der zu 100% kostenlos ist',
+      notificationsTitle: 'Verwalte Benachrichtigungen',
+      notificationsMessage: 'Finde und Verwalte alle deine Benachrichtigungen',
```
Comment (staabm) [inline src/locale/languages/de.js]: `verwalte`

File: src/locale/languages/de.js [hunk 3]
```diff
@@ -0,0 +1,347 @@
...
+      reposTitle: 'Repositorien und Benutzer',
```
Comment (staabm) [inline src/locale/languages/de.js]: würde man eher `Repositories` sagen?

File: src/locale/languages/de.js [hunk 4]
```diff
@@ -0,0 +1,347 @@
...
+      reposMessage:
+        'Erhalte Information über Repositorien, Benutzer und Organisationen',
+      issuesTitle: 'Issues und Pull Requests',
+      issuesMessage:
+        'Unterhalte dich in Konversationen, Merge Pull Requests und vieles mehr',
```
Comment (staabm) [inline src/locale/languages/de.js]: `merge`

File: src/locale/languages/de.js [hunk 5]
```diff
@@ -0,0 +1,347 @@
...
+      signInButton: 'EINLOGGEN',
```
Comment (staabm) [inline src/locale/languages/de.js]: `ANMELDEN` ?

File: src/locale/languages/de.js [hunk 6]
```diff
@@ -0,0 +1,347 @@
...
+    },
+    welcome: {
+      welcomeTitle: 'Willkommen bei GitPoint',
+    },
+    events: {
+      welcomeMessage:
+        'Willkommen! Das ist dein News-Feed - er hälft dich auf dem neusten Stand zu den Benutzern und Repositorien denen du folgst.',
+      commitCommentEvent: 'commented on commit',
```
Comment (staabm) [inline src/locale/languages/de.js]: `kommentierte den Commit` ?

File: src/locale/languages/de.js [hunk 7]
```diff
@@ -0,0 +1,347 @@
...
+      createEvent: 'hat {{object}} erstellt',
+      deleteEvent: 'hat {{object}} gelöscht',
+      issueCommentEvent: '{{action}} on {{type}}',
```
Comment (staabm) [inline src/locale/languages/de.js]: hmm keine idee..

File: src/locale/languages/de.js [hunk 8]
```diff
@@ -0,0 +1,347 @@
...
+      issueEditedEvent: '{{action}} sein Kommentar bei {{type}}',
```
Comment (staabm) [inline src/locale/languages/de.js]: `den Kommentar` ?

File: src/locale/languages/de.js [hunk 9]
```diff
@@ -0,0 +1,347 @@
...
+      issueRemovedEvent: '{{action}} sein Kommentar bei {{type}}',
+      issuesEvent: '{{action}} Issue',
+      publicEvent: {
+        action: 'hat',
+        connector: 'veröffentlicht',
+      },
+      pullRequestEvent: '{{action}} Pull Request',
+      pullRequestReviewEvent: '{{action}} pull request review',
```
Comment (staabm) [inline src/locale/languages/de.js]: `Pull Request`

File: src/locale/languages/de.js [hunk 10]
```diff
@@ -0,0 +1,347 @@
...
+        pullRequest: 'Pull Request',
+        issue: 'Issue',
+      },
+      objects: {
+        repository: 'Repositorium',
+        branch: 'Branch',
+        tag: 'Tag',
+      },
+      actions: {
+        added: 'added',
```
Comment (staabm) [inline src/locale/languages/de.js]: `hinzugefügt`

File: src/locale/languages/de.js [hunk 11]
```diff
@@ -0,0 +1,347 @@
...
+      issueEditedEvent: '{{action}} sein Kommentar bei {{type}}',
+      issueRemovedEvent: '{{action}} sein Kommentar bei {{type}}',
+      issuesEvent: '{{action}} Issue',
+      publicEvent: {
+        action: 'hat',
+        connector: 'veröffentlicht',
+      },
+      pullRequestEvent: '{{action}} Pull Request',
+      pullRequestReviewEvent: '{{action}} pull request review',
+      pullRequestReviewCommentEvent: 'hat eine Pull Request {{action}}',
```
Comment (staabm) [inline src/locale/languages/de.js]: `eine` -> `ein `

File: src/locale/languages/de.js [hunk 12]
```diff
@@ -0,0 +1,347 @@
...
+      codePushChecking: 'Suche nach Updates...',
+      codePushUpdated: 'App ist aktuell!',
+      codePushAvailable: 'Ein Update ist verfügbar!',
+      codePushNotApplicable: 'Nicht verfügbar im Debug Modus',
+    },
+    userOptions: {
+      donate: 'Spenden',
+      title: 'Optionen',
+      language: 'Sprache',
+      privacyPolicy: 'Privacy Policy',
```
Comment (staabm) [inline src/locale/languages/de.js]: `Datenschutzerklärung`

File: src/locale/languages/de.js [hunk 13]
```diff
@@ -0,0 +1,347 @@
...
+      signOut: 'Ausloggen',
```
Comment (staabm) [inline src/locale/languages/de.js]: Abmelden

File: src/locale/languages/de.js [hunk 14]
```diff
@@ -0,0 +1,347 @@
...
+    },
+    privacyPolicy: {
+      title: 'Privacy Policy',
```
Comment (staabm) [inline src/locale/languages/de.js]: Datenschutzerklärung

File: src/locale/languages/de.js (hunk 2 above)
Comment (FlorianTaut) [reply src/locale/languages/de.js]: @staabm Wieso? Es ist ein Titel, der Satz fängt dort an 😆

File: src/locale/languages/de.js (hunk 3 above)
Comment (FlorianTaut) [reply src/locale/languages/de.js]: @staabm Habe da drüber viel nachgedacht, Repositorien ist der korrekte deutche Begriff obwohl es niemand sagt, lässt sich im Nachhinein ja noch anpassen. 

Jetzt bitte wieder englisch, damit uns alle verstehen.

File: src/locale/languages/de.js (hunk 2 above)
Comment (staabm) [reply src/locale/languages/de.js]: `Finde und verwalte` hatte ich gemeint (eine zeile weiter unten)

File: src/locale/languages/de.js (hunk 3 above)
Comment (staabm) [reply src/locale/languages/de.js]: discussing german terms in english is strange and english speaking persons could not help in this discussion nevertheless..

Comment (staabm) [reply src/locale/languages/de.js]: `Repositorien` sounds a bit scientific, never read it anywhere to be honest (except glossaries)

**Comment by FlorianTaut** [comment]
@housseindjirdeh Hey, this can be merged after #459! Thank you!

File: src/locale/languages/de.js (hunk 3 above)
Comment (FlorianTaut) [reply src/locale/languages/de.js]: @staabm Yes, its the official translation, no one would ever say it 😆

**Comment by FlorianTaut** [comment]
@staabm I added a new commit with some improvements you recommended, please check it out and review 👍

**Pull request merged**